libs/community/langchain_community/vectorstores/mongodb_atlas.py

```diff
@@ -32,6 +33,11 @@
 DEFAULT_INSERT_BATCH_SIZE = 100
 
 
+@deprecated(
+    since="0.1.8",
+    removal="0.2.0",
+    alternative_import="langchain_mongodb_atlas.MongoDBAtlasVectorSearch",
+)
 class MongoDBAtlasVectorSearch(VectorStore):
```

Comment:
It's not ideal to duplicate this whole class implementation. Instead can we make this a thin wrapper around langchain_mongodb_atlas.MongoDBAtlasVectorSearch?

Reply:
I don't think we'll be able to do that because the partner-package ends up [needing a completely separate pip installation to use](https://python.langchain.com/docs/contributing/integrations#partner-packages). I imagine having to import `langchain_mongodb_atlas` introduces a breaking change for folks using this class.

Reply:
Ah I see, so the duplication is unavoidable in this case?

Reply:
Yeah, unless @efriis has any suggestions?